stratum/hal/bin/barefoot/docker/start-stratum-container.sh

```diff
@@ -5,7 +5,8 @@ set -e
 
 LOG_DIR=${LOG_DIR:-/var/log}
 SDE_VERSION=${SDE_VERSION:-9.3.2}
-DOCKER_IMAGE=${DOCKER_IMAGE:-stratumproject/stratum-bf}
+#FIXME update "stratum-bf" to "stratum-bfrt" globally
```

Comment:
make this a separate PR

Reply:
Done: #789

Reply:
Thanks Max and Brian for this quick support!

Reply:
thanks!